### PR TITLE
Touch tooltip

### PR DESCRIPTION
**Pull request by vxsx**
if you tapped on a plugin it would stop propagating to the body to "pick" touch version of the plugin. This PR does a workaround + makes sure that we pick the tooltip only once, so there's not much of a performance overhead

### CODE REVIEW

File: cms/static/cms/js/modules/cms.tooltip.js [hunk 1]
```diff
@@ -41,36 +41,43 @@ var CMS = window.CMS || {};
...
-                            generic.eq(0).trigger('dblclick.cms');
-                        }
-                    });
+            /**
+             * Force tooltip to be "touch".
+             *
+             * @method _forceTouch
+             * @private
+             */
+            _forceTouch: function () {
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.tooltip.js]: missing _forceTouch in function

File: cms/static/cms/js/modules/cms.tooltip.js [hunk 2]
```diff
@@ -41,36 +41,43 @@ var CMS = window.CMS || {};
...
             },
 
             /**
              * Checks for touch event and switches to touch tooltip if detected.
              *
-             * @method checkTouch
+             * @method _checkTouch
              * @private
              */
             _checkTouch: function () {
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.tooltip.js]: missing _checkTouch in function

File: cms/static/cms/js/modules/cms.base.js [hunk 3]
```diff
@@ -520,6 +520,30 @@ var CMS = {
...
+             */
+            once: function once(fn) {
+                var result;
+                var didRunOnce = false;
+
+                return function () {
+                    if (!didRunOnce) {
+                        didRunOnce = true;
+                        result = fn.apply(this, arguments);
+                    }
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.base.js]: why not just an else?

File: cms/static/cms/js/modules/cms.tooltip.js (hunk 2 above)
Comment (vxsx) [inline cms/static/cms/js/modules/cms.tooltip.js]: we're actually missing it everywhere in this file

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**